spec/controllers/books_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
```

Comment:
Here you should rename this file to books_controller_spec.rb or books_controller.rb